### PR TITLE
Changes for logging distributed events to app insights using operatio…

### PR DESCRIPTION
**Pull request by fareedmohammed**
…n id and message header correlation id.

https://eaflood.atlassian.net/browse/PSD-921

### CODE REVIEW

File: app/util/app-insights-util.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const appInsights = require('applicationinsights')
+
+module.exports = {
+  setSessionId: (correlationId) => {
+    if (appInsights.defaultClient !== null && appInsights.defaultClient !== undefined) {
```
Comment (johnwatson484) [inline app/util/app-insights-util.js:5]: Do we need to check for `null` too or would `undefined` be sufficient?  It what scenario would it be `null` as opposed to `undefined`?

**Review by johnwatson484 — COMMENTED** [review]
Looks good, just a comment over checking for null and undefined

File: app/util/app-insights-util.js (hunk 1 above)
Comment (fareedmohammed) [reply app/util/app-insights-util.js]: Removed null check as undefined is sufficient.

**Review by johnwatson484 — APPROVED** [review]
LGTM

**Pull request merged**